docker-rpmbuild.sh

```diff
@@ -1,19 +1,28 @@
 #!/bin/bash
+set -e
+cd $(dirname $0)/install
+if [ -e pkg ]; then 
+  mkdir pkg
+fi
 
-SLURM22="22.05.3"
-SLURM20="20.11.9"
+TMP_BINS=slurm-pkgs-tmp
+rm -rf $TMP_BINS
+mkdir $TMP_BINS
 
-rm -rf blobs-build-tmp
-mkdir blobs-build-tmp
 if command -v docker; then
+  extra_args=
+  if command -v podman; then
+    extra_args="--privileged"
+  fi
 
-  docker run -v $(pwd)/specs/default/cluster-init/files:/source -v $(pwd)/blobs-debian-tmp:/root/rpmbuild/RPMS/x86_64 -ti almalinux:8.5 /bin/bash /source/00-build-slurm.sh centos $SLURM22
-  docker run -v $(pwd)/specs/default/cluster-init/files:/source -v $(pwd)/blobs-debian-tmp:/root/rpmbuild/RPMS/x86_64 -ti centos:7 /bin/bash /source/00-build-slurm.sh centos $SLURM20
-  docker run -v $(pwd)/specs/default/cluster-init/files:/source -v $(pwd)/blobs-debian-tmp:/root/rpmbuild/RPMS/x86_64 -ti ubuntu:18.04 /bin/bash /source/01-build-debs.sh
-  # we only want the rpms for almalinux8 (i.e. slurm22) but we want all of the debs
-  mv blobs-debian-tmp/*.deb blobs/
-  mv blobs-debian-tmp/*$SLURM22*.rpm blobs/
-  rm -rf blobs-debian-tmp
+  docker run -v $(pwd)/specs/default/cluster-init/files:/source -v $(pwd)/$TMP_BINS:/root/rpmbuild/RPMS/x86_64 $extra_args -ti almalinux:8.5 /bin/bash /source/00-build-slurm.sh centos
+  docker run -v $(pwd)/specs/default/cluster-init/files:/source -v $(pwd)/$TMP_BINS:/root/rpmbuild/RPMS/x86_64 -ti centos:7 /bin/bash /source/00-build-slurm.sh centos
+  docker run -v $(pwd)/specs/default/cluster-init/files:/source -v $(pwd)/$TMP_BINS:/root/rpmbuild/RPMS/x86_64 $extra_args -ti ubuntu:18.04 /bin/bash /source/01-build-debs.sh
+  
+  mv $TMP_BINS/* slurm-pkgs/
```

Comment:
This line failed for me: since slurm-pkgs directory was not there already. I think it expects the directory to be pre-made.

`mv: target 'slurm-pkgs/' is not a directory`